tests/unit/general/test_array.cpp

```diff
@@ -18,8 +18,7 @@ TEST_CASE("Array init-list construction", "[Array]")
 {
    int ContigData[6] = {6, 5, 4, 3, 2, 1};
    Array<int> a(ContigData, 6);
-   Array<int> b({6.0, 5.0, 4.0, 3.0, 2.0, 1.0});
-
+   Array<int> b{6.0, 5.0, 4.0, 3.0, 2.0, 1.0};
```

Comment:
Instead of replacing one style of object construction with another, it is best to test them both.